pkg/cmd/roachprod-microbench/compare.go

```diff
@@ -20,25 +21,39 @@ import (
 	"sort"
 	"strings"
 	"text/template"
+	"time"
 
 	"github.com/cockroachdb/cockroach/pkg/cmd/roachprod-microbench/google"
 	"github.com/cockroachdb/cockroach/pkg/cmd/roachprod-microbench/model"
+	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
 	"github.com/cockroachdb/errors"
 	"github.com/cockroachdb/errors/oserror"
+	influxdb2 "github.com/influxdata/influxdb-client-go/v2"
 	"github.com/slack-go/slack"
 	"golang.org/x/exp/maps"
 	"golang.org/x/perf/benchfmt"
+	"golang.org/x/perf/benchseries"
 )
 
 type compareConfig struct {
-	newDir             string
-	oldDir             string
-	sheetDesc          string
-	slackUser          string
-	slackChannel       string
-	slackToken         string
-	threshold          float64
-	publishGoogleSheet bool
+	slackConfig
+	influxConfig
```

Comment:
Nit: might be a good idea to not embed these structs, as referencing their fields directly from `compareConfig` is confusing / ambiguous.

Reply:
Good point, I'll name these.